harness/determined/common/util.py

```diff
@@ -28,6 +31,12 @@
 
 _yaml = yaml.YAML(typ="safe", pure=True)
 
+MetricType = str
+Metrics = Dict[str, Any]
+LEGACY_TRAINING = "training"  # no need to call these legacy_..
```

Comment:
also, these might belong more in `harness.determined.util`?

Reply:
I don't know. these are not super clear to me. I can move them if you think they belong there.